src/main.cpp

```diff
@@ -172,6 +166,7 @@ typedef struct {
 
 static Model model;
 static Model *g = &model;
+std::list<Explosive> explosivesList;
 
 int gettestFlag(){                  
```

Comment:
Is there anything else you could name this function and the variable testFlag? It's the exact same as a variable in the test.cpp file, but I feel like that may be more appropriately named. It's obviously a small critique but I wanted to point it out.